### ISSUE TITLE
Surface flux tallies

### ISSUE THREAD

**Issue by paulromano** [openmc-dev/openmc#2]

**Comment by ternence6** [comment]
I wonder what I should do to transfer the flux scores' units into the common units——neutrons per centimeter squared per second.

**Comment by paulromano** [comment]
@ternence6 We recently added a section in the user's guide about normalizing units:
https://docs.openmc.org/en/latest/usersguide/tallies.html#normalization-of-tally-results

**Comment by ternence6** [comment]
@paulromano ty 👍

**Comment by gonuke** [comment]
Is it correct that surface flux tallies remain unimplemented?

**Comment by paulromano** [comment]
@gonuke Yes, it is possible to tally surface currents, but surface fluxes remain to be implemented.

**Comment by GuySten** [comment]
I've opened PR #3742 to implement surface flux tallies.
